samples/SmokeTest/SmokeTest/BlobStorageTest.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Azure.Storage.Blobs;
+using System;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices.ComTypes;
+using System.Threading.Tasks;
+
+namespace SmokeTest
+{
+
+
+    class BlobStorageTest : TestBase
+    {
+        
+        private BlobServiceClient service;
+        private BlockBlobClient blob;
+
+        public BlobStorageTest(string connectionString,string containerName, string blobName)
+        {
+            this.service = new BlobServiceClient(connectionString);
+            this.blob = service.GetBlobContainerClient(containerName).GetBlockBlobClient(blobName);
+        }
+
+        /// <summary>
+        /// Test the Storage Blobs SDK
+        /// </summary>
+        /// <returns>true if passes, false if fails</returns>
+        public async Task<bool> RunTests()
+        {
+            Console.WriteLine("\n---------------------------------");
+            Console.WriteLine("STORAGE");
+            Console.WriteLine("---------------------------------");
+            Console.WriteLine("Functionalities to test: 2:");
+            Console.WriteLine("1.- Upload Blob Block");
+            Console.WriteLine("2.- Delete that Blob Block" + '\n');
+
+            Console.Write("Uploading blob... ");
+            var result1 = await ExcecuteTest(UploadBlob);
+            if (!result1)
+            {
+                //If this test failed, then the othe one is going to fail too.
+                Console.WriteLine("Cannot delete the Blob.");
```

Comment:
I'd suggest you passing in the message to ExecuteTest method and show it in the exception handler in that method.

Reply:
Resolving this review since the high order function is not going to be used anymore.